### PR TITLE
Update deploying-native-app.md

### PR DESCRIPTION
**Pull request by aramlawi**

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@aramlawi please sign the CLA.

File: content/howto/mobile/deploying-native-app.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ If your `build` command fails citing version conflicts on Java classes, try the
 1. Clear your deployment directory.
 2. Complete a new build in Studio Pro.
 3. Run the `build` command again in the Native Builder.
+4. Compare your Java Built path with the version used in Studio Pro
```
Comment (jogboms) [inline content/howto/mobile/deploying-native-app.md]: This is indeed a specific case if the client has multiple versions of java installed on their machine regardless of Studio pro.

**Comment by ConnorLand** [comment]
Once you sign the CLA @aramlawi I can merge. Thank you for contributing to the documentation!

**Comment by aramlawi** [comment]
CLA should be signed now.

Op do 9 jan. 2020 om 15:12 schreef ConnorLand <notifications@github.com>:

**Pull request merged**